install.md

```diff
@@ -29,18 +29,11 @@ pip install jupyterlab
 
 If installing using `pip install --user`, you must add the user-level `bin` directory to your `PATH` environment variable in order to launch `jupyter lab`.
 
-## Getting started with the classic Jupyter Notebook
-
-### Prerequisite: Python
-
-While Jupyter runs code in many programming languages, Python is a requirement
-(Python 3.3 or greater, or Python 2.7) for installing the JupyterLab or the classic Jupyter Notebook.
-
-### Installing Jupyter Notebook using Conda
+## Getting started with the Classic Notebook
```

Comment:
```suggestion
## Getting started with the classic Jupyter Notebook
```
"classic" I think is always an adjective, not actually part of the title, as it is used below.